### ISSUE TITLE
Notification setting (notification_id) not working for iOS

### ISSUE THREAD

**Issue by flivni** [guyromb/cordova-open-native-settings#44]
I'm using `openNativeSettings.open('notification_id')` to open the notification settings page for my app.

This works on Android but doesn't work for on iOS. I'm running
* iOS 12.1.4.
* cordova-open-native-settings 1.5.2

I took a look at the source code and read this issue:
https://github.com/guyromb/Cordova-open-native-settings/issues/21

It seems to me that for this to work on iOS we need to append `&path=APP_BUNDLE_ID` to the url. But I don't see this is happening in the source code. 

Thanks for the help!

**Comment by fuyucn** [comment]
its not work for no reason. I use "application_details" to work around this, and works fine.

**Comment by a-ridley** [comment]
@fuyucn @flivni Using "application_details" will only take you to the apps general settings. There is 1 more tap required to reach the apps notification settings. Did you settle with only reach the apps general settings?

I also noticed some apps are getting [binary rejections](https://githubmemory.com/repo/guyromb/cordova-open-native-settings/issues?cursor=Y3Vyc29yOnYyOpK5MjAxOC0xMS0wOVQxNTozNTo0MyswODowMM4Wl-Vo&pagination=next&page=3) from apple due to using 'prefs:root' or 'App-Prefs:root'. Although this plugin doesn't necessarily use either of those with the word root it does use 'App-Prefs:' for the prefix. I am not familiar with objective-c but I am concerned that root can be appended to that prefix consequently causing apps to run the risk of getting rejected.

**Comment by Sampath-Lokuge** [comment]
Yes, this is still not working with iOS 14.7.1 too. But `application_details` at least move into the particular app's content rather than a generic/all app content view.
